mne/io/meas_info.py

```diff
@@ -1513,3 +1505,32 @@ def _force_update_info(info_base, info_target):
             continue
         for i_targ in info_target:
             i_targ[key] = val
+
+
+def anonymize_info(info):
+    """Anonymize measurement information in place by removing 'subject_info',
+       'meas_date', 'file_id', 'meas_id' if they exist.
```

Comment:
Should be a single line, so you can put the actual fields below on a separate line if you want